README.md

```diff
@@ -1 +1,92 @@
-# school-library
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Vet clinic database
```

Comment:
- [ ]  **[OPTIONAL]** Kindly change your Project name to be relevant to this project.

Reply:
Hi TedLivist,
Thanks for the suggestion.